public/js/domBoking.js

```diff
@@ -13,5 +22,22 @@ formContactUs.addEventListener('submit', (e) => {
     body: JSON.stringify(dataContactUs),
     headers: { 'Content-Type': 'application/json' },
   })
+    .then(res => res.json())
+    .then((res) => {
+      if (res.msg === 'done') {
+        popup.classList.add('popup-show');
+        popupMassage.textContent = 'We received your message, we will send you our response soon.';
+      }
```

Comment:
I think we should add a `else { throw new Error("error in fetch response");}` or something like that to get into the catch block after